bin/docserv

```diff
@@ -241,8 +244,8 @@ class DocConfParser:
         # then read all files into an xml tree
         self.tree = ElementTree.parse(tmp_filename)
         xml_root = self.tree.getroot()
-        self.branch = xml_root.find(".//productindex[@id='%s']/docset[@id='%s']/builddocs/language[@code='%s']/branch" % (self.build_instruction['product'], self.build_instruction['docset'], self.build_instruction['lang'])).text
-        self.remote_repo = repo = xml_root.find(".//productindex[@id='%s']/docset[@id='%s']/builddocs/git/remote" % (self.build_instruction['product'], self.build_instruction['docset'])).text
+        self.branch = xml_root.find(".//product[@productid='%s']/docset[@setid='%s']/builddocs/language[@lang='%s']/branch" % (self.build_instruction['product'], self.build_instruction['docset'], self.build_instruction['lang'])).text
```

Comment:
Hmn, if it is feasible, we should avoid the descendant axis (`.//`). You can try the `.findall` method without `.//`. The arguments are the same.